TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>This takes an "object_name" argument, so we can pass "Trappist-1" as an input without needing to input the coordinates ourselves (<a href="https://astroquery.readthedocs.io/en/latest/mast/mast.html#tesscut" target="_blank">tesscut readthedocs</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>